src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,71 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Support for several Java 8 API added.
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8 by default
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate c.g.g.validation
```

Comment:
JSR 303 Bean Validation on client side is one of the "killerfeatures" of GWT for me. There are missing features in GWT (especially because of missing reflections) and no progress since GWT 2.5 but works in general. I've tried to fix the missing features in my [gwt-bean-validators](https://github.com/ManfredTremmel/gwt-bean-validators) project and I'm very interested in keeping this feature. I could to take over the validation feature into my project or contribute to the gwtproject itself.
At end of July I could invest some time to work into the topic. Let me know if you are interested.